docs/days/day-10.md

```diff
@@ -316,7 +316,7 @@ class CategoryController extends AbstractController
 }
 ```
 
-That’s it. Now admin can create as much categories as wants.
+That’s it. Now admin user can create as much categories as she wants.
```

Comment:
`she`? We can't link user to specific gender here.

Reply:
well, I was referring to the female gender on purpose. We live in a post-feminism world where 
 a good rule of thumb  is to refer to the user implying feminine gender. : )

`as wants` sounds not very good to me, idk. May be it should be changed to `as admin user wants` ?

Reply:
Use "as one wants" or "as they want". I asked Kate.

Reply:
Also please fix "many categories", not **much**.

Reply:
hah, awesome, sure, thanks